### ISSUE TITLE
[proposal] Extract DateTools into separate library.

### ISSUE THREAD

**Issue by nitrobin** [hyperfiction/HypSystem#26]
We are using HypSystem in our projects for monitoring network status and device information. 

It works very well, thanks! :)

But after long time app evolution on android and update to latest android-supported-libs we was faced with "64K Reference Limit" https://developer.android.com/studio/build/multidex.html?hl=ru#about
After investigating I was found that app has many unused joda-time classes.
We was cut libs with "jarjar" and fully remove Joda-time and DateTools.hx in "vendoring" version of HypSystem and it works now. 
It would be great to be able option to turn off the Joda-time dependency at compile time or split lib.

I think splitting into two libs (device information and date tools) will be more practical, thanks!

**Comment by mastef** [comment]
Yes, I agree, in the lime build process all the resources always get copied over which makes pre-dexing of joda-time also impossible slowing down the android build process. Hypsystem is great for device information, +1 for moving DateTools into separate library
